src/common/system.rs

```diff
@@ -1887,6 +1888,7 @@ impl ProcessRefreshKind {
             environ: UpdateKind::OnlyIfNotSet,
             cmd: UpdateKind::OnlyIfNotSet,
             exe: UpdateKind::OnlyIfNotSet,
+            thread: false,
```

Comment:
Should be `true`.

Reply:
is that because it was true before ? Isnt it better to keep it consistent with Windows version where it doesnt provide thread details at all. Linux provides it as an optional feature. Also green engineering :)

Reply:
for now left it as default true.